client/payments/payment-recommendations.tsx

```diff
@@ -221,9 +221,9 @@ const PaymentRecommendations: React.FC = () => {
 					/>
 				</div>
 			</CardHeader>
-			<CardBody>
+			<div>
```

Comment:
I think we could skip this `div` to avoid an extra tag.  Thoughts?

Reply:
>I think we can also remove the body padding changes from payment-recommendations.scss.

I was thinking to preserve the structure but I don't think it matters in this case, removed in 54bd850. I've also removed the CSS